packages/protocol/contracts/common/EssentialContract.sol

```diff
@@ -22,8 +22,8 @@ abstract contract EssentialContract is UUPSUpgradeable, Ownable2StepUpgradeable,
         0xa5054f728453d3dbe953bdc43e4d0cb97e662ea32d7958190f3dc2da31d9721b;
 
     /// @dev Slot 1.
-    uint8 private __reentry;
-    uint8 private __paused;
+    uint8 public __reentry;
```

Comment:
instead of making these public:

- change `_inNonReentrant` to `inNonReentrant` and make it public
- we already have `paused` as a public method